zebra/interface.c

```diff
@@ -639,6 +639,22 @@ static void if_install_connected(struct interface *ifp)
 	}
 }
 
+/* Interface has connected routes ? */
+bool if_has_connected(struct interface *ifp, afi_t afi)
```

Comment:
nit: has connected what? IMO, it would be better **if_has_connected_routes** then?

Reply:
done